src/block-components/button/attributes.js

```diff
@@ -0,0 +1,51 @@
+import { addLinkAttributes } from '../helpers/link'
+import { addBorderAttributes } from '../helpers/borders'
+
+export const addAttributes = ( attrObject, options = {} ) => {
+	const {
+		selector = '.stk-button__button',
+	} = options
+	attrObject.add( {
+		attributes: {
+			buttonPadding: {
+				stkResponsive: true,
+				type: 'object',
+			},
+			buttonPaddingUnit: {
+				stkResponsive: true,
+				type: 'string',
+				default: 'px',
+			},
+			buttonBackgroundColorType: {
+				stkHover: true,
+				type: 'string',
+				default: '',
+			},
+			buttonBackgroundColor: {
+				stkHover: true,
+				type: 'string',
+				default: '', // button primary color.
+			},
+			buttonBackgroundColor2: {
+				stkHover: true,
+				type: 'string',
+				default: '',
+			},
+			buttonBackgroundGradientDirection: {
+				stkHover: true,
+				type: 'number',
+				default: '',
+			},
+			text: {
```

Comment:
Since this attribute is used primarily for buttons, please use `buttonText` instead of button to prevent conflicts in with other block components

Reply:
The `text` attribute should not be included in the button block-component. I will remove this.